package.json

```diff
@@ -2,32 +2,33 @@
   "name": "kalpa-tree",
   "version": "3.4.0",
   "description": "An efficient hierarchical tree for the dom",
-  "main": "index.js",
+  "node": ">=14.16",
   "dependencies": {
     "d3-drag": "^1.0.1",
     "d3-selection": "^1.0.2",
     "d3-transition": "^1.0.3",
     "escape-string-regexp": "^1.0.3",
-    "node-sass": "^4.9.0",
+    "sass": "^1.51.0",
     "scoreboard-colors": "git+https://github.com/SpiderStrategies/scoreboard-colors.git#v4.2.0"
   },
   "devDependencies": {
-    "JSONStream": "^1.0.6",
     "browserify": "^12.0.1",
     "browserify-css": "^0.8.1",
     "double-tap": "^1.0.2",
+    "esmify": "^2.1.1",
+    "JSONStream": "^1.0.6",
     "request": "^2.36.0",
     "smokestack": "^3.4.1",
     "tape": "^3.5.0",
     "watchify": "^3.6.1"
   },
   "scripts": {
-    "bundle": "browserify -r ./index.js:tree -r http -r JSONStream > example/bundle.js -d",
-    "css": "node scripts/css.js",
+    "bundle": "browserify -p esmify -r ./index.js:tree -r http -r JSONStream > example/bundle.js -d",
```

Comment:
is this just an intermediate step?  I thought yesterday you were leaning towards making kalpa-tree et al. ESM and making Impact deal with the esmify concern?

Reply:
@codecounselor This is on my list of things to talk you about on slack. 

This project is still using browserify for development. Once we confirm our theories discussed yesterday, I'll take the time to migrate this to esbuild or vite. But this little `esmify` allows local development (and testing) to stay the same for now as we're experimenting.